Makefile

```diff
@@ -56,6 +56,9 @@ build: ## Create wheel distribution
 .PHONY: deploy
 deploy: ## Deploy wheel to public s3 bucket
 	aws s3 cp --recursive --exclude "*" --include "*.whl" --acl public-read dist/ s3://$(PUBLIC_PROJECT_BUCKET)
+	git tag v$(shell python setup.py --version)
```

Comment:
This is great, I really like tying the release to a tag. This way we version control the data too because of dvc, so we can have the cake and it eat it too.

Reply:
To be honest I am new to releases and tags. What is the alternative? To tag manually when you remember?

Reply:
No, I think this is best practice. What I meant was that in your case there is a bonus because data is also versioned through GitHub, so a tag will also point to a specific version of the data.

Potentially we can think of versioning models too through tags, like spacy does: https://github.com/explosion/spacy-models/releases.

This way a release tag encompasses everything, model, code and (through dvc.lock) data.

Reply:
Well dvc also versions models through `dvc repro` so if you go to a previous commit the model is rolled to whatever version was produced then

Reply:
Ah, yeah, that's right, even better.